### PR TITLE
FIX

### PR DESCRIPTION
**Pull request by luigimolinaro**

### CODE REVIEW

**Comment by fketelaars** [comment]
Two comments:
- Please add documentation to explain how an existing ROSA cluster can be used
- Please follow steps here: https://ibm.github.io/cloud-pak-deployer/80-development/deployer-development-setup/#set-up-gpg-for-the-development-user to sign off your commits.

**Comment by luigimolinaro** [comment]
> Two comments:
> 
> * Please add documentation to explain how an existing ROSA cluster can be used
> * Please follow steps here: https://ibm.github.io/cloud-pak-deployer/80-development/deployer-development-setup/#set-up-gpg-for-the-development-user to sign off your commits.

Done.

**Comment by fketelaars** [comment]
I see the PR is still in draft. Do want to merge it?

**Review by fketelaars — CHANGES_REQUESTED** [review]
Please add documentation here: https://github.com/IBM/cloud-pak-deployer/blob/main/docs/src/10-use-deployer/3-run/aws-rosa.md

**Comment by luigimolinaro** [comment]
> Please add documentation here: https://github.com/IBM/cloud-pak-deployer/blob/main/docs/src/10-use-deployer/3-run/aws-rosa.md

Done

File: automation-roles/99-generic/openshift/openshift-login/tasks/aws-login-rosa-ocp.yml [hunk 1]
```diff
@@ -28,10 +28,19 @@
     name: vault-get-secret
   vars:
     secret_name: "{{ _p_openshift_cluster_name }}-cluster-admin-password"
-    secret_group: "{{ environment_name }}" 
+    secret_group: "{{ environment_name }}"
+  failed_when: vault_get_secret.rc != 0
```
Comment (fketelaars) [inline automation-roles/99-generic/openshift/openshift-login/tasks/aws-login-rosa-ocp.yml:32]: Please confirm that you tested this. So far I know the vault-get-secret role does not return a rc element. To validate whether the secret was existing, you have to check that the secret_value is blank.

**Review by fketelaars — CHANGES_REQUESTED** [review]
See inline comment about the rc for vault-get-secret.

File: automation-roles/99-generic/openshift/openshift-login/tasks/aws-login-rosa-ocp.yml (hunk 1 above)
Comment (luigimolinaro) [reply automation-roles/99-generic/openshift/openshift-login/tasks/aws-login-rosa-ocp.yml:33]: I added a modification checks if secret_value is defined and not empty (secret_value != '') before setting the _rosa_cluster_admin_password fact. This ensures that the fact is only set if a non-blank value was retrieved from the vault.

**Review by fketelaars — APPROVED** [review]

**Pull request merged**